### ISSUE TITLE
Quest: Mutiny on the Mercy

### ISSUE THREAD

**Issue by Ergates82** [dalaranwow/dalaran-wow#5080]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The mutinous sea dogs that you have to kill in order to loot the gunpowder from are not below deck, and do not spawn in regardless of time waited, rendering the quest uncompletable.

**Expected behaviour**: Tell us what should happen instead
The mutinous sea dogs should be below the deck of the ship.

**Steps to reproduce the problem**:

1. Board the Sister Mercy
2. Go below deck
3. No ghouls will be there.

**Include proofs for this behaviour**
http://www.wowhead.com/quest=11527/mutiny-on-the-mercy

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Duplicate with #457
